apps/user_ldap/lib/User_LDAP.php

```diff
@@ -175,16 +179,11 @@ public function getLDAPUserByLoginName($loginName) {
 	 * @return false|string
 	 */
 	public function checkPassword($uid, $password) {
-		try {
-			$ldapRecord = $this->getLDAPUserByLoginName($uid);
-		} catch (NotOnLDAP $e) {
-			$this->logger->debug(
-				$e->getMessage(),
-				['app' => 'user_ldap', 'exception' => $e]
-			);
+		$username = $this->loginName2UserName($uid, true);
+		if (!$username) {
```

Comment:
```suggestion
		if ($username === false) {
```
To avoid trouble if username is `'0'`

Reply:
Good idea :+1: 

Fixed @come-nc